task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -0,0 +1,60 @@
+import time
+from threading import Thread
+
+from pyrsistent import m
+from six.moves.queue import Queue
+
+from task_processing.interfaces.task_executor import TaskExecutor
+
+
+class RetryingExecutor(TaskExecutor):
```

Comment:
it's a matter of conveniece? the argument "why provide `function` if user can just do `implementation body`" always rests on a tradeoff between flexibility and convenience.